sdcm/stress_thread.py

```diff
@@ -329,8 +329,8 @@ def _run_cs_stress(self, loader, loader_idx, cpu_idx, keyspace_idx):  # pylint:
                 hdr_logger_context:
             publisher.event_id = cs_stress_event.event_id
             try:
-                # prolong timeout by 5% to avoid killing cassandra-stress process
-                hard_timeout = self.timeout + int(self.timeout * 0.05)
+                # prolong timeout by 10% to avoid killing cassandra-stress process
+                hard_timeout = self.timeout + int(self.timeout * 0.1)
```

Comment:
we can extend hard_timeout,

but the errors we see are cause of the soft_timeout, i.e. the timeout being passed to SoftTimeoutContext

we should give self.time more %5, and the hard_timeout more 10%

Reply:
added soft_timeout